diff --git a/node_modules/libsignal/src/session_record.js b/node_modules/libsignal/src/session_record.js
index 7626a39..c80d5d0 100644
--- a/node_modules/libsignal/src/session_record.js
+++ b/node_modules/libsignal/src/session_record.js
@@ -267,18 +267,12 @@ class SessionRecord {
 
     closeSession(session) {
         if (this.isClosed(session)) {
-            console.warn("Session already closed", session);
             return;
         }
-        console.info("Closing session:", session);
         session.indexInfo.closed = Date.now();
     }
 
     openSession(session) {
-        if (!this.isClosed(session)) {
-            console.warn("Session already open");
-        }
-        console.info("Opening session:", session);
         session.indexInfo.closed = -1;
     }
 
@@ -298,7 +292,6 @@ class SessionRecord {
                 }
             }
             if (oldestKey) {
-                console.info("Removing old closed session:", oldestSession);
                 delete this.sessions[oldestKey];
             } else {
                 throw new Error('Corrupt sessions object');
